examples/green/Dockerfile

```diff
@@ -75,6 +75,8 @@ COPY --from=toolkit /usr/bin/elemental /usr/bin/elemental
 RUN systemctl enable NetworkManager.service && \
     systemctl enable sshd.service
 
+RUN useradd --system systemd-timesync
```

Comment:
I wonder if there should be a `system-user-*` package for that, I tried to find one and I could not. `system-user-ntp` does not provide it either.